UI/window-basic-main-transitions.cpp

```diff
@@ -1621,3 +1637,34 @@ int OBSBasic::GetOverrideTransitionDuration(OBSSource source)
 
 	return (int)obs_data_get_int(data, "transition_duration");
 }
+
+void OBSBasic::SetPreviewProgramIndicators()
+{
+	if (!IsPreviewProgramMode())
+		return;
+
+	for (int i = 0; i < ui->scenes->count(); i++) {
+		QListWidgetItem *item = ui->scenes->item(i);
+		OBSScene scene = GetOBSRef<OBSScene>(item);
+
+		obs_source_t *source = obs_scene_get_source(scene);
+
+		if (source == GetProgramSource()) {
+			item->setIcon(
+				QIcon(":res/images/program-indicator.svg"));
+		} else if (source == GetCurrentSceneSource()) {
+			item->setIcon(
+				QIcon(":res/images/preview-indicator.svg"));
```

Comment:
I feel the colours should be configurable by the theme at least. Could do something similar to the tray icon where it checks if the theme has a definition first.

Reply:
They are using the same colors as the multiview.

Reply:
Multiview uses `#d00000` and `#00d000` rather than `#ff0000` and `#00ff00` (at least based on how they're rendered)

Reply:
Changed to the correct colors.